File: sk000801/입문/공 던지기.java
// k번째 2*(k-1)
class Solution {
    public int solution(int[] numbers, int k) {
        if(numbers.length > 2*(k-1)) return numbers[2*(k-1)];
        else {
            int[] many = new int[2*k];
            for(int i = 0; i < 2*k; i++) {
                many[i] = numbers[i%numbers.length];
            }
            return many[2*(k-1)];
        }
    }
}

File: kyw0716/level0/공 던지기.js
function solution(numbers, k) {
  let index = 0;
  let preIndex = index;

  while (k > 0) {
    preIndex = index;
    index += 2;
    index = index % numbers.length;
    k -= 1;
  }

  return numbers[preIndex];
}

// https://school.programmers.co.kr/learn/courses/30/lessons/120843
